src/DynamoWPFCLI/GeometryHolder.cs

```diff
@@ -9,27 +10,180 @@
 
 namespace DynamoWPFCLI
 {
+    /// <summary>
+    /// The class that represents data for drawing a graphic primitive 
+    /// </summary>
+       
+    internal class GraphicPrimitives
```

Comment:
@sm6srw what is the difference between this and https://github.com/DynamoDS/Dynamo/blob/dec6240ded0c4369617775336b9af60c2aba4103/src/NodeServices/GraphicsDataInterfaces.cs#L10

Reply:
It is a version of the data in the IRenderPackage but the types and method names are different. I borrowed this class from Reach and then updated it to follow the naming Convention used by CoGs for easy serialization to JSON. Do you think it is worth doing this differently? Add a ToJson method to iRenderPackage?

Reply:
Not sure if it would work, but you could inherit from renderPackage and then use JsonProperty attributes to serialize into the format.

Or could add a new interface backing this object which is the same as the format.

I think keeping it internal for now is fine but we would want to expose creation methods for it eventually, and not just through the CLI but as part of some kind of export from dynamoCore or as part of DynamoServices I think using the render package for this keeps it simpler, but not sure if that works for you or if you think it's better to have some indirection between the output and render packages. Just curious on your approach.

Reply:
Thanks, I will take a look at both options.